README.md

```diff
@@ -1,3 +1,32 @@
 # adapt-navigationTitle
 
-AT Compatible
+**Navigation Title** is an *extension* for displaying a title in the course navigation bar. The title can be hidden for mobile view.
+
+<img src='https://user-images.githubusercontent.com/898168/210417005-3c2f0e9d-b1f0-4a7b-815c-33c3a6921965.jpg' width="500" alt="Screenshot">
+
+### Attributes
+
+The following attributes are set within *course.json*.
+
+**\_navigationTitle** (object): The course.json Navigation Title target attribute object.
+
+>**\_isEnabled** (boolean): Controls whether the Navigation Title course object is enabled or not.
+
+>**\_hideForMobile** (boolean): Optional, hide title for mobile view. Useful to declutter the navigation bar for mobile view where limited space is available.
+
+>**\_useCourseTitle** (boolean): Optional, use the course title as the navigation title. Will override anything in the `title` property of the `_navigationTitle` object.
+
+>**title** (string): The title text to display.
+
+## Limitations
+
+No known limitations
+
+----------------------------
+**Version number:**  1.0.0  
```

Comment:
Could we change the double spacing at the end of the version number and subsequent lines to a html <br> tag instead to avoid the line breaks being removed when white space is removed.

Reply:
Done.